### ISSUE TITLE
[BUG] 最近几天第一帧似乎吃不到ControlNET，不知道是哪边的问题

### ISSUE THREAD

**Issue by neavo** [OedoSoldier/enhanced-img2img#30]
![image](https://user-images.githubusercontent.com/998676/234448695-6f88ff72-3cc4-4675-96f5-f3b8202ecaa9.png)

如图，应该区别很明显了

脚本跑出来的第一帧，相同参数下手动跑出来的第一帧完全不一样，但是第二帧开始就正常了

不知道是ContrlNET更新的问题还是脚本的问题

**Comment by openwaterer** [comment]
是的 很明显，你第一帧是透明通道的原图，第二帧才开始图image2image,生成了黑色背景。我准备手动把第一帧替换成image2image生成的图来解决这个问题
